### PR TITLE
docs: typo

### PR DESCRIPTION
**Pull request by thecodesalim**
Fixed a type from an unique to a unique

### CODE REVIEW

File: packages/docs/core-concepts/index.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
-Before diving into core concepts, we need to know that a store is defined using `defineStore()` and that it requires an **unique** name, passed as the first argument:
+Before diving into core concepts, we need to know that a store is defined using `defineStore()` and that it requires a **
+** name, passed as the first argument:
 
 ```js
 import { defineStore } from 'pinia'
 
 // useStore could be anything like useUser, useCart
-// the first argument is a unique id of the store across your application
+// the first argument is a
```
Comment (posva) [inline packages/docs/core-concepts/index.md]: can you remove this change

File: packages/docs/core-concepts/index.md [hunk 2]
```diff
@@ -1,12 +1,17 @@
 # Defining a Store
 
-Before diving into core concepts, we need to know that a store is defined using `defineStore()` and that it requires an **unique** name, passed as the first argument:
+Before diving into core concepts, we need to know that a store is defined using `defineStore()` and that it requires a **
+** name, passed as the first argument:
```
Comment (posva) [inline packages/docs/core-concepts/index.md]: the word ended up missing

**Review by posva — CHANGES_REQUESTED** [review]
Can

File: packages/docs/core-concepts/index.md (hunk 1 above)
Comment (thecodesalim) [reply packages/docs/core-concepts/index.md]: done

File: packages/docs/core-concepts/index.md (hunk 2 above)
Comment (thecodesalim) [reply packages/docs/core-concepts/index.md]: sorry for the mistake, i have fixed it

File: packages/docs/core-concepts/index.md [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 
-Before diving into core concepts, we need to know that a store is defined using `defineStore()` and that it requires an **unique** name, passed as the first argument:
+Before diving into core concepts, we need to know that a store is defined using `defineStore()` and that it requires a **unique** name, passed as the first argument:
 
 ```js
 import { defineStore } from 'pinia'
 
 // useStore could be anything like useUser, useCart
-// the first argument is a unique id of the store across your application
+// the first argument is a uique id of the store across your application
```
Comment (posva) [inline packages/docs/core-concepts/index.md]:
```suggestion
// the first argument is a unique id of the store across your application
```

**Review by posva — APPROVED** [review]

**Pull request merged**